.github/deploy.sh

```diff
@@ -25,16 +25,18 @@ if [[ $BETA = "true" ]]; then
 fi
 
 # Generate version index that is shown as root index page
-cp util/gh-pages/versions.html out/index.html
-
-echo "Making the versions.json file"
-python3 ./util/versions.py out
+python3 ./util/versions.py ./util/gh-pages/versions.html out
 
 # Now let's go have some fun with the cloned repo
 cd out
 git config user.name "GHA CI"
 git config user.email "gha@ci.invalid"
 
+git status
+
+# Remove old versions.json, it's no longer used
+git rm --ignore-unmatch versions.json
+
```

Comment:
This seems unnecessary. This is probably a one-time commit once this PR is merged. No need to include this in the script.